backend/handlers/handlers.go

```diff
@@ -242,7 +242,8 @@ func NewHandlers(
 	getAPIRouterNoError(apiRouter)("/exchange/by-region/{code}", handlers.getExchangesByRegion).Methods("GET")
 	getAPIRouterNoError(apiRouter)("/exchange/deals/{action}/{code}", handlers.getExchangeDeals).Methods("GET")
 	getAPIRouterNoError(apiRouter)("/exchange/supported/{code}", handlers.getExchangeSupported).Methods("GET")
-	getAPIRouterNoError(apiRouter)("/exchange/btcdirect-otc/supported/{code}", handlers.getExchangeBtcDirectOTCSupported).Methods("GET")
+	getAPIRouterNoError(apiRouter)("/exchange/btcdirect/supported/{code}", handlers.getExchangeBtcDirectOTCSupported).Methods("GET")
```

Comment:
Here it should still be `exchange/btcdirect-otc/supported`.
the OTC service of btc-direct is something a bit different from the other widget/deals things